Recon/src/Recon/OverlayProducer.cxx

```diff
@@ -65,8 +65,8 @@ void OverlayProducer::onNewRun(const ldmx::RunHeader &) {
         framework::RandomNumberSeedService::CONDITIONS_OBJECT_NAME);
     rndm_ = std::make_unique<TRandom2>(rnss.getSeed("OverlayProducer::rndm"));
   }
-
-  int start_event = rndm_->Uniform(20., 1e4);
+  // TAV: These boundaries should be configurable
+  int start_event = rndm_->Uniform(20., 4570);
```

Comment:
what's this change about?

Reply:
Made an issue here https://github.com/LDMX-Software/ldmx-sw/issues/1638
we can come back to this in a next PR